airflow/providers/google/cloud/hooks/bigquery.py

```diff
@@ -2390,6 +2397,50 @@ def var_print(var_name):
 
         return project_id, dataset_id, table_id
 
+    @GoogleBaseHook.fallback_to_default_project_id
+    def get_query_results(
+        self,
+        job_id: str,
+        location: str,
+        max_results: int | None = None,
```

Comment:
`max_result_counts` ? `max_results` sounds like `max(results)`

Reply:
I followed the same naming conventions as in the Google API, which are also used in other methods of this class.
I'd rather keep it `max_results` - its meaning should be quite clear.

Reply:
got it. in that case, let's keep be as `max_results`